src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -142,7 +145,7 @@ class AuthScreens extends React.Component {
             if (isLoadingApp) {
                 App.openApp();
             } else {
-                App.reconnectApp(this.props.lastUpdateIDAppliedToClient);
+                App.reconnectApp(this.props.lastUpdateIDAppliedToClient ?? 0);
```

Comment:
hmm this looks suspicious, shouldn we call this function if there isn't `lastUpdateIDAppliedToClient` ?

Reply:
@kubabutkiewicz Maybe, but overall reconnectApp anyway has default updateIDFrom value as 0
![image](https://github.com/Expensify/App/assets/23176449/50dbb7a7-308d-428d-a919-9e9e1d43e56a)